src/info/repo/dependencies.rs

```diff
@@ -0,0 +1,35 @@
+use crate::info::info_field::{InfoField, InfoType};
+use manifest::Manifest;
+
+pub struct DependenciesInfo {
+    pub dependencies: String,
+}
+
+impl DependenciesInfo {
+    pub fn new(manifest: &Option<Manifest>) -> Self {
+        let dependencies = match manifest {
+            Some(m) => {
+                if m.number_of_dependencies == 0 {
```

Comment:
As a fan of `<bool>.then()` I'd do: `(m.number_of_dependencies != 0).then(|| format!(…)).unwrap_or_default()`. Combined with the above I'd get `manifest.and_then(|m| (m.number_of_dependencies != 0).then(|| format!(…))).unwrap_or_default()`.